<html>

<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<title>Motif discovery in plasmodium</title>
</head>


<body>
<p>

<h2>MDOS: a method for Motif Discovery using Orthologous Sequences</h2>

MDOS is a comparative genomics method for regulatory motif
discovery. Different from previous methods, it does not require or
use sequence alignments. It is particularly suitable for discovering
motifs in highly diverged genomes, or genomes with sequence alignments
difficult to obtain or poor in quality. 
<p>

The code is written in Java and can be downloaded from 
 <a href="code/mdos.tar.gz">here</a>.
You will need Java version 1.5 or above to run the code. 



<p>
<p>
<h2>Supplementary Data</h2>

<ul>
<li> <a href="data/orth.pfa_pyo.one2one.up.seq">1Kb upstream sequences of orthologous genes beween
P. falciparm and P. yoelii</a>
<li> <a href="data/orth.pfa_pkn.one2one.up.seq"> 1Kb upstream sequences of orthologous genes beween
P. falciparm and P. knowlesi</a>
<li>  <a href="data/CER_BAY_U.fa">1Kb upstream sequences of orthologous genes between
 S. cerevisiae and S. bayanus</a>
<li>  <a href="data/ELE_BRE_U.fa">1Kb upstream sequences of orthologous genes between C. elegans and
C. briggase</a>
<li>  <a href="data/MEL_PSE_U.fa">2Kb upstream sequences of orthologous genes between D. melanogaster and D. pseudoobscura</a>
</ul>

<h2>Supplementary Tables</h2>
<ul>
<li><a href="tables/pyo.up.cluster..xls">Clusters of motifs discovered
in the upstream regions of P. falciparm and P. yoelii</a>
<li><a href="tables/organism/yeast_7mer_Zscore.xls">List of 7-mers with conservation z-scores in the
upstream regions of  S. cerevisiae and S. bayanus</a>
<li><a href="tables/organism/worm_7mer_Zscore.xls">List of 7-mers with conservation z-scores in the
upstream regions of C. elegans and C. briggase</a>
<li><a href="tables/organism/fly_7mer_Zscore.xls">List of 7-mers with conservation z-scores in the
upstream regions of D. melanogaster and D. pseudoobscura</a>
</ul>


</body>
</html>
